rllib/utils/tests/test_taskpool.py

```diff
@@ -0,0 +1,138 @@
+import unittest
+from unittest.mock import patch
```

Comment:
Note that you should add this to run_rllib_tests.sh

Reply:
Thanks, I wasn't sure if it would be picked up automatically or not, I'll add that in the morning.

Reply:
Thanks, I wasn't sure if it would be picked up automatically or not, I'll add that in the morning.

Reply:
I added these tests to `rllib/BUILD` as I saw this note within the file:
```
# NOTE: Move more test cases here from ci/jenkins/run_rllib_tests.sh
# as Travis tests seem to run more stable.
```
These are also short running tests that don't initialise Ray and so executing them within a Docker container felt like overkill. I'm happy to move these to `run_rllib_tests.sh` if you think that's till the most appropriate place for them.